inc/Modules/Forms/RestApi.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @package Dforms
+ */
+
+namespace Develtio\WP\Forms\Modules\Forms;
+
+use Develtio\WP\Forms\Core\Base\BaseController;
+use Develtio\WP\Hoya\Handlers\Step1FormHandleRequest;
+
+/**
+ * Class RestApi
+ * @package Develtio\WP\Forms\Modules\Forms
+ */
+class RestApi extends BaseController
+{
+    /**
+     * @var string
+     */
+    public $post_type_name;
+
+    /**
+     * @var CreateForm $form_instance
+     */
+    public $form_instance;
+
+    private $custom_handler = null;
```

Comment:
maybe you are able to add  `@var CustomHandlerInterface|null` here

p.s.
I do not know what class it should be so I put there CustomHandlerInterface